### DISCUSSION TITLE
Oranchada doubts

### DISCUSSION THREAD

**Discussion by MariaICV** [h2020charisma/oranchada#16] [Q&A]
We have some doubts about the use of some functions of Oranchada:

FInd peaks: It work but, how can I export the data?:
![image](https://user-images.githubusercontent.com/129367889/228928112-f6bc6379-04db-4e9e-a38e-4d7f5f3e7dc1.png)
Fit peaks: It does not work: After having waited more than 5 minutes, no data is plotted or obtained:
![image](https://user-images.githubusercontent.com/129367889/228928192-3568443f-d40e-45f4-a852-f57d5686416d.png)
Again, despite having waited a few minutes, I cannot see the plotting of any spectrum in the database:
![image](https://user-images.githubusercontent.com/129367889/228928254-b8324dbb-34d6-422d-8cf4-cf4b9e6de097.png)
Xaxis fine calibration: I am able to correct the x axis but is there a way to export this data?
![image](https://user-images.githubusercontent.com/129367889/228928320-8cb66be7-4808-4154-aa6b-29d388c5ad03.png)

Thank you!

**Comment by georgievgeorgi** [comment]
- **Fitting**: The fit should be taking input from find peaks, not from the raw spectrum. Of course this is not yet documented. I tried to provide an example and realized that there is a problem in the code. That problem was most probably introduced during the last modifications that i have done.
- **Test spectra**: I tried to visualize exactly the same spectrum from your example without any problems. I have access only to linux so cannot easily test it on windows. Will investigate with my colleagues with windows.
- **Calibration**: At the moment there is no way to export the calibration from Oranchada. This calibration is a result of iterative process an it is not easy export it the calibration as factors of a polynomial. The calibration is a list of corresponding values -- for each channel number on the x-axlis there is a corresponding Raman shift in $[\mathrm{cm}^{-1}]$. We can easily provide a way to export this export these data to a table or to a CSV-file.

Please let me know it you need to have the calibration information exported or you will be fine to use it on the next steps in Oranchada.
The rest of the problems i need to fix with my IDEA colleagues.
Thank you for attracting my attention to these problems
